src/AwsClient.php

```diff
@@ -582,6 +582,8 @@ private function setClientBuiltIns($args)
             $builtIns['AWS::S3::ForcePathStyle'] = $config['use_path_style_endpoint'];
             $builtIns['AWS::S3::DisableMultiRegionAccessPoints'] = $config['disable_multiregion_access_points'];
         }
+        $builtIns['AWS::Auth::AccountId'] = new AccountIdLazyResolver($this->credentialProvider, $clientConfig['account_id_endpoint_mode']);
+
```

Comment:
Looks like `AWS::Auth::AccountIdEndpointMode` is also considered a builtin per the EP2.0 spec.